scan/scan_tasks.py

```diff
@@ -1,10 +1,8 @@
 # scan_tasks.py
 
 from textwrap import dedent
-
```

Comment:
This looks like formatting wasn't run? I would run ruff format scan and I think this file at least will get reformatted.

Reply:
Ran ```poetry run ruff format scan``` and it fixed it